pcsx2/gui/Debugger/CtrlMemSearch.h

```diff
@@ -0,0 +1,106 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2021  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include <wx/wx.h>
+#include "common/PersistentThread.h"
+#include "DebugTools/DebugInterface.h"
+#include "DebugTools/DisassemblyManager.h"
+
+enum SEARCHTYPE
+{
+	TYPE_BYTE,
+	TYPE_WORD,
+	TYPE_DWORD,
+	TYPE_QWORD,
+	TYPE_FLOAT,
+	TYPE_DOUBLE,
+	TYPE_STRING,
+};
```

Comment:
Since you're referencing it with the namespace anyways, why not make this an `enum class` and then leave out the `TYPE_` prefix?

Reply:
Good point.